pyvista/__init__.py

```diff
@@ -22,7 +22,8 @@
 
 
 # determine if using vtk > 5
-if vtk.vtkVersion().GetVTKMajorVersion() <= 5:
+from pyvista import _vtk
```

Comment:
Move to the top with the rest of the `from pyvista` lines?

Reply:
Yes, this needs to be moved up.  I'll resolve this.